docs/setup/utilizing-grafana.md

```diff
@@ -1,2 +1,90 @@
 # Utilizing Grafana
 
+Grafana is a powerful open-source platform that allows users to visualize
+and monitor various metrics and data sources. In this guide, we will
+explore how to effectively use Grafana to monitor and analyze GitHub Actions.
+
+Grafana can be accessed through
+[the following URL](https://mon.scality.net/grafana/d/WidbLgPnk/).
+Once logged in, you will find two distinct boards: "GitHub Actions Costs"
+and "GitHub Actions Monitoring." These boards provide valuable insights
+into cost monitoring and general information about GitHub Actions.
+
+## As a user
+
+When accessing the Grafana URL, users can easily sort and filter the displayed
+data using variables located at the top of the page. This functionality
+allows more precise and targeted information based on specific criteria.
+
+Available Sorting Options:
+
+- `repository`: GitHub repository
+- `workflow_name`: Name of the workflow
+- `repository_visibility`: Public or private
+- `job_name`: Name of the job
+- `runner_type`: GitHub or self-hosted
+- `cloud`: GCloud / AWS or GitHub
```

Comment:
cloud only exist because that's how we have configured the instance. It should be removed from this project.

Reply:
Done